packages/grid/x-grid/src/tests/apiRef.XGrid.test.tsx

```diff
@@ -154,7 +154,13 @@ describe('<XGrid /> - apiRef', () => {
 
   it('getDataAsCsv should return the correct string representation of the grid data', () => {
     render(<TestCase />);
```

Comment:
I think that we should refactor the least possible in the tests. DRY is our enemy. How about we render just the data we need instead of calling `updateRows`?

Reply:
No preference. I followed the approach done for the other tests and the `TestCase` does work for that use case.

Reply:
I think that providing props for the rows would already be better than using the `apiRef.current.updateRows` API. The more declartive we can get (avoiding imperative calls) and multiple renders, the better.

Reply:
Done, I split the test case and added 2 more.

Reply:
Thanks. Speaking of abstraction and duplication in the tests. I suspect that we could abstract the usage of the apiRef as well as the container dimensions. We will see.